src/server/components/dashboard/listsItems/controllers.ts

```diff
@@ -717,8 +717,11 @@ export async function listItemEditRequestValidation(
     return next(err);
 
   } else if (!userIsListPublisher(req, list)) {
-    const err = new HttpException(403, "403", "User does not have publishing rights on this list.");
-    return next(err);
+    res.status(403).send({
```

Comment:
Ha it's been so long I can't fully remember.I think it was something to do with returning a 403 page with an error message or something. So since it lands on a 403 error page there's no need for it to pass the error to the next piece of middleware.